### PR TITLE
Disabling use of rancher-metadata FQDN

### PR DESCRIPTION
**Pull request by leodotcloud**
https://github.com/rancher/rancher/issues/9521

Also the ipsec image has fixes for issue: https://github.com/rancher/rancher/issues/8204

### CODE REVIEW

**Comment by niusmallnan** [comment]
@leodotcloud 
RANCHER_METADATA_ADDRESS or RANCHER_METADATA_IP? A typo?

**Comment by leodotcloud** [comment]
Thank you @niusmallnan, it's a typo.

File: infra-templates/healthcheck/4/rancher-compose.yml [hunk 1]
```diff
@@ -2,7 +2,14 @@
   name: "Healthcheck Service"
   version: "v0.3.3"
   description: "Rancher healthcheck service"
-  minimum_rancher_version: v1.6.8-rc1  
+  minimum_rancher_version: v1.6.8-rc1
+  questions:
+    - variable: "RANCHER_METADATA_ADDRESS"
+      label: "Rancher Metadata Address"
+      description: "Rancher Metadata Address"
```
Comment (deniseschannon) [inline infra-templates/healthcheck/4/rancher-compose.yml]: Can we give something more descriptive? And is it really configurable?

**Comment by deniseschannon** [comment]
@leodotcloud

healthcheck: please add in the changelog of what was changed. 

vxlan: there shouldn't be a new folder for vxlan as we didn't introduce it into the latest rancher release. please update folder 11 with your changes.

The max rancher version for previous versions (so old folders) should be checked to make sure it says `v1.6.7`

File: infra-templates/vxlan/11/README.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ Traffic to and from hosts requires UDP port `4789` to be open.
 
 #### Router and CNI Driver [rancher/net:v0.11.6]
```
Comment (deniseschannon) [inline infra-templates/vxlan/11/README.md]: update to `rancher/net:v0.11.8`

Comment (deniseschannon) [inline infra-templates/vxlan/11/README.md]: also, we should update min version for this folder to `v1.6.8-rc1`, and update max version in folder 10 to `v1.6.7`. the current min/max is residual from the last release attempt.

**Comment by deniseschannon** [comment]
LGTM

**Pull request merged**